great_expectations/exceptions/exceptions.py

```diff
@@ -48,6 +48,11 @@ class ExpectationSuiteError(DataContextError):
     pass
 
 
+class ExpectationSuiteSaveError(ExpectationSuiteError):
+    def __init__(self, name: str) -> None:
+        super().__init__(message=f"Could not save ExpectationSuite '{name}'")
```

Comment:
I don't know if I love these error types - could use suggestions on naming! Are they too granular?

Reply:
We are suppose to err on the side of granularity and let people doing the error aggregation cloud feature group as needed.